etc/errors/anvil.json

```diff
@@ -1115,7 +1251,57 @@
                   }
                 ]
               }
+            },
+            {
+              "name": "SerializationError",
+              "code": 30,
+              "message": "Failed to parse L1 transaction from request (from={from}, to={to}): {reason}.",
```

Comment:
Is it truly L1 transaction or is this error for any kind of transaction?

Reply:
Can be both. Fixed.